roofit/roofitcore/inc/RooVectorDataStore.h

```diff
@@ -77,7 +78,14 @@ class RooVectorDataStore : public RooAbsDataStore {
     std::size_t size;
   };
 
+  /// \name Internal RooFit interface.
```

Comment:
Move this up before `struct ArraysStruct`?

Reply:
Ignore this. 🙂
You can only "hide" this if you put it in
```c++
namespace RooFit { namespace Internal { ... } }
```
or something like that, but you have to define it outside of the class in this case.

Reply:
I would have liked to, but one can't group member classes like member functions or data members unfortunately... I tried it out, bit it had no effect on how the doxygen was rendered. So I thought it's better to not put it in the group so nobody is mislead to thing that the `struct ArraysStruct` appears in the "Internal RooFit interface".

Reply:
That's true, good idea! Or maybe we can even follow the convention of the rest of ROOT here and use `ROOT::Detail`.

But in this special case with the RooVectorDataStore, I will leave it as a class in the class because then it's consistent with the already existing internal helper classes `RealVec`, `RealFullVec`, etc.